doc/man/8/ceph-objectstore-tool.rst

```diff
@@ -0,0 +1,492 @@
+:orphan:
+
+=============================================================
+ceph-objectstore-tool - modify or examine the state of an OSD
+=============================================================
+
+Synopsis
+========
+
+
+| **ceph-objectstore-tool** --data-path *path to osd* [--op *list* ]
+
+
+
+Possible object operations:
```

Comment:
I would move this to the description section below:

**ceph-objectstore-tool** --data-path *path to osd* [ --op *list $OBJECT_ID*]

Would also change that usage to:

**ceph-objectstore-tool** --data-path *path to osd* op_list_json|objectID object_operation